unity/Assets/Scripts/AgentManager.cs

```diff
@@ -471,7 +488,9 @@ private enum serverTypes { WSGI, FIFO };
         float fieldOfView = DEFAULT_FOV,
         string skyboxColor = null,
         bool orthographic = false,
-        float? orthographicSize = null
+        float? orthographicSize = null,
+        float? nearClippingPlane = null,
+        float? farClippingPlane = null
```

Comment:
It looks like these do not need to be optional `float?`, since they are being parsed as:
```c#
        if (farClippingPlane == null) {
            camera.farClipPlane = 20.0f;
        } else {
            camera.farClipPlane = (float) farClippingPlane;
        }
```
can:

- `float? farClippingPlane = null` become `float farClippingPlane = 20.0f`
- `float? nearClippingPlane = null` become `float nearClippingPlane = 0.1f`

Reply:
ok check the new commit, made changes